### PR TITLE
pyfatx: Fix assertion failure on extraction of empty file; bump version to 0.0.7

### PR DESCRIPTION
**Pull request by antangelo**

### CODE REVIEW

File: pyfatx/__init__.py [hunk 1]
```diff
@@ -148,6 +148,9 @@ def read(self, path: str, offset: int = 0, size: Optional[int] = None) -> bytes:
 		path = self._sanitize_path(path)
 		attr = self.get_attr(path)
 		assert(attr.is_file)
+		if (offset == 0 and attr.file_size == 0):
```
Comment (abaire) [inline pyfatx/__init__.py:151]: Nit: Drop the parenthesis (unnecessary as the conditional fits on one line).

File: pyfatx/__init__.py [hunk 2]
```diff
@@ -148,6 +148,9 @@ def read(self, path: str, offset: int = 0, size: Optional[int] = None) -> bytes:
...
+			return bytes()
```
Comment (abaire) [inline pyfatx/__init__.py:152]: Nit, it uses `b''` on line 158, should probably be consistent in how we return empty buffers.

File: pyfatx/__init__.py (hunk 1 above)
Comment (antangelo) [reply pyfatx/__init__.py]: Fixed, muscle memory from too much C programming

File: pyfatx/__init__.py (hunk 2 above)
Comment (antangelo) [reply pyfatx/__init__.py]: Fixed

**Review by abaire — APPROVED** [review]

**Pull request merged**